src/test/isolation2/input/local_directory_table_mixed.source

```diff
@@ -0,0 +1,131 @@
+-- Mixed test for local directory table
+-- setup for temp tablespace
+! rm -rf '@testtablespace@';
+! mkdir -p '@testtablespace@';
+CREATE TABLESPACE directory_tblspc LOCATION '@testtablespace@';
+
+CREATE EXTENSION IF NOT EXISTS gp_inject_fault;
+
+-- Test create directory table and create directory table
+1: BEGIN;
+2: BEGIN;
+1: CREATE DIRECTORY TABLE dir_table1 TABLESPACE directory_tblspc;
+2: CREATE DIRECTORY TABLE dir_table2 TABLESPACE directory_tblspc;
+1: COMMIT;
+2: COMMIT;
+
+1: COPY BINARY dir_table1 FROM '@abs_srcdir@/data/nation.csv' 'nation1';
+2: COPY BINARY dir_table2 FROM '@abs_srcdir@/data/nation.csv' 'nation2';
+3: COPY BINARY dir_table1 FROM '@abs_srcdir@/data/nation.csv' 'nation3';
+4: COPY BINARY dir_table2 FROM '@abs_srcdir@/data/nation.csv' 'nation4';
+
+-- Test copy binary from the same directory table in parallel session is allowed
+1: BEGIN;
+2: BEGIN;
+1: COPY BINARY dir_table1 FROM '@abs_srcdir@/data/nation.csv' 'nation5';
+2: COPY BINARY dir_table1 FROM '@abs_srcdir@/data/nation.csv' 'nation6';
+1: COMMIT;
+2: COMMIT;
+
+-- Test copy binary from and select directory table
+1: BEGIN;
+2: BEGIN;
+1: COPY BINARY dir_table2 FROM '@abs_srcdir@/data/nation.csv' 'nation5';
+2: SELECT relative_path, size, tag, md5 FROM dir_table2 ORDER BY 1;
+1: COMMIT;
+2: COMMIT;
+
+1: BEGIN;
+2: BEGIN;
+1: COPY BINARY dir_table2 FROM '@abs_srcdir@/data/nation.csv' 'nation6';
```

Comment:
seem to it can copy `/usr/local/bin/xx` to a table ?

Reply:
Yes, only when has the file privilege, it's permitted to do this.

Reply:
gpadmin has the read privilege for /home/gpadmin/xx, seem to maybe leak user's private file.

Reply:
Yes, this can't be checked in database side, now if database user has the privilege of the file, we should allow it to put file into directory table.